ArrayFire/ArrayFire.sln

```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
```

Comment:
These solution/project files are for VS2015. Which VS2015 Community is available I would recommend using VS2013 sln/projects. That way users who want to upgrade to 2015 can do so while users with 2013 should not face any issues.